Use FileInfo.Length instead of MemoryStream.Length

Ky7m opened the pull request:
This pull request add a small fix to avoid object allocation of MemoryStream on getting information about file size. /cc @DamianEdwards

Commit:
Use FileInfo.Length instead MemoryStream.Length (avoid object allocation of MS)

Commit:
Use FileInfo.Length instead MemoryStream.Length (avoid object allocation of MS)

dnfclas commented:
Hi __@Ky7m__, I'm your friendly neighborhood .NET Foundation Pull Request Bot (You can call me DNFBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, DNFBOT;

DamianEdwards committed:
Tweaks to AppInsights dependency tracking

DamianEdwards commented:
I'm this case I actually want to create the stream as that forces the file contents to be cached, which is the whole point of this method :smile:

Ky7m commented:
Got it. It makes perfect sense now.

DamianEdwards committed:
Minor update

DamianEdwards committed:
More AppInsights tweaks

DamianEdwards committed:
More AppInsights tracking tweaks.
Now tracking the show start/stop as events.

DamianEdwards committed:
Use the GIT commit SHA as the AppInsights application version

DamianEdwards committed:
More AppInsights tweaks

DamianEdwards committed:
More AppInsights tweaks & some refactoring:
- Track environment name in all telemetry events
- Move cached web root setup into its own service extension method
- Move non-HTTP app start (Configure) into an IStartupFilter (AppStart)
- Track AppInsight event for app startup & shutdown
- Add JS to track authenticated users

Commit:
Revert "Use FileInfo.Length instead MemoryStream.Length (avoid object allocation of MS)"

This reverts commit 3c9d324fc75d9cc6c17733d9e837c6f900b435c1.

Commit:
Merge branch 'dev' of https://github.com/Ky7m/live.asp.net into dev
